### ISSUE TITLE
Bug with if else if, main function is never found.

### ISSUE THREAD

**Issue by nibblebits** [nibblebits/DragonCompiler#46]
```

#define __WORDSIZE 32
#if __WORDSIZE == 32

#elif __WORDSIZE == 64

#else
# error "abc"
#endif

int main()
{
   return 55;
}

```

**Comment by nibblebits** [comment]
Issue is on the else if keyword #elif

**Comment by nibblebits** [comment]
Fixed: 51399c34cbcd533e89777ed9a36d132b8f35ef06
